docker/README.md

```diff
@@ -0,0 +1,10 @@
+#Docker
+To build a docker image for `tyk-sync`, run:
+```
+docker build -t tyk-sync:latest .
+```
+To run `tyk-sync` run
+```
+docker run -it --network="host" -v $(pwd)/apis:/apis --rm tyk-sync:latest tyk-sync dump -s <admin_secret> -d https://tyk-dashboard.test-backend.vdbinfra.nl/ -t /apis
```

Comment:
Can you please change it to <api-access-token>? (not to be confused with the admin secret that is used for `/admin/{endpoints}`)